server/webhook_parser.go

```diff
@@ -394,14 +394,24 @@ func mergeWebhookEvents(events []*webhook) Webhook {
 
 	for _, event := range events {
 		merged.eventTypes = merged.eventTypes.Union(event.eventTypes)
+
+		fromWithDefault := event.fieldInfo.from
```

Comment:
nit/naming: suggestion s/fromWithDefault/from/?

Reply:
I was staying consistent with what's going on [here](https://github.com/mattermost/mattermost-plugin-jira/blob/8e9304f9acc722fb519135155c2cf9f2e3408738/server/webhook_parser.go#L93). No problem changing to `from`.

Though this points out that the fields that do not have the `fromWithDefault` assigned in the link I have above (`from` would be is assigned the empty string) are the ones that have the issue. So the code I've introduced is kind of a cleanup for the fields left out in the initial parse, so the issue will still occur on a "one field" changelog event. @cpoile thoughts on changing the initial parsing code instead of this section?

Reply:
1/5 `from` is sufficiently intention-revealing, 3/5 it looks typographically more pleasing, and there's no conflict within the scope

Reply:
Yah, that webhook parser from/toWithDefault could be simplified too. And it looks like the cleanup can be moved to the parser... 

Why didn't I change all the `from` parameters to `fromWithDefault`? That doesn't make sense... Then we wouldn't be having this trouble in the first place.
Sorry about that.

Reply:
Oh, I think I remember -- there were two PRs right around the same time, I think I changed all the from/to parameters to from/toWithDefault, and then a day later added the from and to fields to the rest of the calls (and forgot that they should be changed to WithDefaults, because that first PR hadn't been merged yet)... That might explain it.

Reply:
@cpoile It was my PR earlier that had the issue :slightly_smiling_face: 

Basically any call to `parseWebhookUpdatedField` should pass in the defaults, as that function is meant for generic usage.